docs/inspector/ux-ui-playbook.md

```
@@ -0,0 +1,163 @@
+export const useTraceStore = create<TraceSlice>()(
+  devtools(
+    (set, get) => ({
+      traces: new Map<string, ParsedTrace>(),
+      addTrace: (sid, trace) =>
+        set(state => ({ traces: new Map(state.traces).set(sid, trace) }))
+    }),
+    { name: 'trace-store', serialize: { map: true } }
+  )
+)
+5 · Performance rules
+Concern	Budget	Technique
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 8

(MD010, no-hard-tabs)